### PR TITLE
troubleshooting-kubeadm: add guide for fixing stale CoreDNS pods

### PR DESCRIPTION
**Pull request by neolit123**
fixes kubernetes/kubeadm#998

/assign @chrisohaver 
/assign @Bradamant3 
@kubernetes/sig-cluster-lifecycle-pr-reviews 

preview:
https://deploy-preview-9872--kubernetes-io-master-staging.netlify.com/docs/setup/independent/troubleshooting-kubeadm/#coredns-pods-have-crashloopbackoff-or-error-state

### CODE REVIEW

File: content/en/docs/setup/independent/troubleshooting-kubeadm.md [hunk 1]
```diff
@@ -238,4 +238,23 @@ EOF
...
+- Disable SELinux.
+- Modify the `coredns` deployment to set `allowPrivilegeEscalation` to `true`:
+
+```bash
+kubectl -n kube-system get deployment coredns -o yaml | \
+  sed 's/allowPrivilegeEscalation: false/allowPrivilegeEscalation: true/g' | \
+  kubectl apply -f -
+```
+
+Please mind that disabling SELinux or setting `allowPrivilegeEscalation` to `true`
```
Comment (Bradamant3) [inline content/en/docs/setup/independent/troubleshooting-kubeadm.md]: Let's get rid of "please" (not relevant here, way overused generally throughout these docs) and make the whole thing stronger:

{{< warning >}}
**Warning:**  Disabling SELinux or setting `allowPrivilegeEscalation` to `true` can compromise the security of your cluster.
{{< /warning >}}

**Comment by neolit123** [comment]
@Bradamant3 updated.

**Comment by Bradamant3** [comment]
@kubernetes/sig-cluster-lifecycle-pr-reviews if someone can lgtm this for technical accuracy, we'll get it merged, thanks.

/approve for doc review

**Comment by chrisohaver** [comment]
Should we make note of versions of docker that we know work (and/or cause problems).
For example, version 1.13 has this problem, where as 17.03 does not.

Or do we want to steer clear of naming specific versions.

**Comment by chrisohaver** [comment]
LGTM - Perhaps we should indicate that the first option is recommended.

**Comment by neolit123** [comment]
@chrisohaver 
> Should we make note of versions of docker that we know work (and/or cause problems).
For example, version 1.13 has this problem, where as 17.03 does not.

the linked guide recommends 17.03 and hopefully later version don't have the problem too.

> LGTM - Perhaps we should indicate that the first option is recommended.

well. there are 3 options, two of them are marked with a warning. so with the process of elimination the first one is the recommended one. isn't that sufficient?

**Comment by chrisohaver** [comment]
OK - lgtm

File: content/en/docs/setup/independent/troubleshooting-kubeadm.md [hunk 2]
```diff
@@ -238,4 +238,25 @@ EOF
...
-{{% /capture %}}
\ No newline at end of file
+
+## `coredns` pods have `CrashLoopBackOff` or `Error` state
+
+If you have nodes that are running SELinux with an older version of Docker you might experience a scenario
+where the `coredns` pods are not starting. To solve that you can try one of the following options:
+
+- Upgrade to a [newer version of Docker](/docs/setup/independent/install-kubeadm/#installing-docker).
+- Disable SELinux.
```
Comment (dixudx) [inline content/en/docs/setup/independent/troubleshooting-kubeadm.md]: Shall we add some commands or links here to guide users how to disable this?

* Edit the `/etc/selinux/config` file to set the  `SELINUX` parameter to  `disabled`, and then reboot the server.
* Run the `setenforce 0` command.

Or add such a link on [disabling selinux](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/security-enhanced_linux/sect-security-enhanced_linux-enabling_and_disabling_selinux-disabling_selinux).

Comment (neolit123) [reply content/en/docs/setup/independent/troubleshooting-kubeadm.md]: sounds good. :+1: 
i will add the link later today.

**Comment by neolit123** [comment]
made small update to turn `Disable SELinux` into an external link.

**Pull request merged**